runtime/src/main/cpp/KString.cpp

```diff
@@ -734,12 +736,12 @@ OBJ_GETTER(Kotlin_String_replace, KString thiz, KChar oldChar, KChar newChar, KB
   KChar* resultRaw = CharArrayAddressOfElementAt(result, 0);
   if (ignoreCase) {
     KChar oldCharLower = towlower_Konan(oldChar);
-    for (KInt index = 0; index < count; ++index) {
+    for (uint32_t index = 0; index < count; ++index) {
```

Comment:
[Minor] Again, it's a bit susceptible regarding (nvisible) type of count_.
`decltype(thiz->count_)` is exact but noisy. Other variants may be simple `while (count--)` or possibly less fragile `for (auto end = thizRaw + thiz->count_; thizRaw < end; )`

Reply:
Let's postpone for now, answered [here](https://github.com/JetBrains/kotlin-native/pull/4319#discussion_r468648150)